packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -5,21 +5,43 @@
 
 import React from "react";
 
-import { ITelemetryBaseEvent } from "@fluidframework/common-definitions";
 import {
+	handleIncomingMessage,
+	ISourcedDebuggerMessage,
 	InboundHandlers,
 	TelemetryHistoryMessage,
-	GetTelemetryHistoryMessage,
 	TelemetryEventMessage,
-	ISourcedDebuggerMessage,
-	handleIncomingMessage,
-	postMessagesToWindow,
 } from "@fluid-tools/client-debugger";
+import { ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+import { useMessageRelay } from "../MessageRelayContext";
+
+function mapEventCategoryToBackgroundColor(eventCategory: string): string {
+	switch (eventCategory) {
+		case "generic":
+			return "#b8ebf2";
```

Comment:
Probably not for this PR, but I wonder if we should be mapping to FluentUI theme values here, rather than explicit colors.

Reply:
We totally should. We'll want to make that part of the final visual homologation (@rohandubal, @WayneFerrao).